profiles/kentik_snmp/cisco/traps.yml

```diff
@@ -0,0 +1,102 @@
+# Cisco SNMP traps
+---
+
+traps:
+  # Generic SNMP traps
+  # https://oidref.com/static/circitor/SNMPv2-MIB.mib
+  - trap_oid: 1.3.6.1.6.3.1.1.5.1
+    trap_name: coldStart
+    drop_undefined: false
+  - trap_oid: 1.3.6.1.6.3.1.1.5.2
+    trap_name: warmStart
+    drop_undefined: false
+  - trap_oid: 1.3.6.1.6.3.1.1.5.3
+    trap_name: linkDown
+    drop_undefined: false
+  - trap_oid: 1.3.6.1.6.3.1.1.5.4
+    trap_name: linkUp
+    drop_undefined: false
+  - trap_oid: 1.3.6.1.6.3.1.1.5.5
+    trap_name: authenticationFailure
+    drop_undefined: false
```

Comment:
curious: why would we not put these into a `/_general/traps.yml` profile?

Reply:
Looking at the customer data where I got this from the trap OID is generic, but its also included varbinds that are specific to Cisco mibs.  I had the same thought until I saw that Cisco specific stuff.  I think its similar in concept to the way the cisco entity sensor mib gets all mingled with the generic snmp entity mibs.